frontend/src/views/Instructors/InstructorsList/InstructorsList.tsx

```diff
@@ -36,12 +91,20 @@ export const InstructorsList = () => {
   if (instructorsError || qualificationsError) {
     return <GeneralAPIError />;
   }
-
-  if (instructorsData === undefined || qualificationsData === undefined) {
+  if (qualificationsData === undefined) {
     return <FullPageLoading />;
   }
 
-  const rows = instructorsData.instructors;
+  const qualificationsOptions: PicklistOption[] = [];
+  qualificationsData.categories.forEach((quali) => {
+    qualificationsOptions.push({ value: quali.id, label: quali.name });
+  });
+
+  const rows = instructorsData?.instructors;
+  const totalRows = instructorsData?.total ?? 0;
```

Comment:
Let's wrap the total in the `useListTotal` hook.
All you have to do is change this into
```js
  const totalRows = useListTotal(instructorsData?.total)
```
and put it before any returns